src/main/java/org/openstreetmap/atlas/generator/AtlasGeneratorHelper.java

```diff
@@ -444,14 +323,14 @@ protected static AtlasLoadingOption buildAtlasLoadingOption(final CountryBoundar
                 final String country = countryShard.getCountry();
                 final Set<Shard> possibleShards = getAllShardsForCountry(tasks, country);
 
-                logger.info("Started sectioning raw Atlas for {}", countryShardString);
-
+                // Instantiate the cache
+                final HadoopAtlasFileCache atlasCache = new HadoopAtlasFileCache(slicedRawAtlasPath,
+                        sparkContext);
                 // Create the fetcher
                 final Function<Shard, Optional<Atlas>> slicedRawAtlasFetcher = AtlasGeneratorHelper
-                        .atlasFetcher(SparkFileHelper.combine(slicedRawAtlasPath, country),
-                                System.getProperty("java.io.tmpdir"), country, sparkContext,
-                                possibleShards);
+                        .atlasFetcher(atlasCache, country, possibleShards);
 
+                logger.info("Started sectioning raw Atlas for {}", countryShardString);
```

Comment:
Nit: Sorry, you'll hate me for this - can you remove this please? The WaySectionProcessor has very fine-grained logging already and logs this exact line at it's level.

Reply:
np, sane logging is important!